WalletWasabi/Crypto/Api/CredentialIssuer.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using WalletWasabi.Crypto.Groups;
+using WalletWasabi.Crypto.Randomness;
+using WalletWasabi.Crypto.ZeroKnowledge;
+using WalletWasabi.Crypto.ZeroKnowledge.LinearRelation;
+using WalletWasabi.Crypto.ZeroKnowledge.NonInteractive;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Crypto.Api
+{
+	public class CredentialIssuer
```

Comment:
What is your suggestion? I like `CredentialIssuer` a lot btw, it makes perfect sense to me.

Reply:
i think it might be confusing if we have two sets of issuer parameters per issuer, and also because the registration actions are higher level than credential issuance, since they deal with bitcoin concepts (inputs, outputs)... let's keep it for now but maybe consider extracting the credential specific details into a delegate later, after which we might want to rename (e.g. `Coordinator` w/ 2 `Issuer` sub objects)